web/json_options.go

```diff
@@ -20,20 +20,46 @@ import "time"
 // JSONOption values passed to ConvertJSONObjectList.
 type jsonOptions struct {
 	complexAttributeNameDelimiter string
-	dateTimeFormats               []string
+	dateTimeFormats               []DateTimeFormatWithTz
```

Comment:
This change will not be backwards compatible, but I felt this was the cleanest way to implement this. If you're opposed to releasing a new major version (assuming we're following semantic versioning), I can take a different approach and split this out into 2 field (e.g. `dateTimeFormats` and `dateTimeFormatsNoTz`)

Reply:
No problem with breaking compatibility at this point.